### PR TITLE
fix: keep packageurl-js at an older version

### PR DESCRIPTION
**Pull request by SteveShani**
- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?
Upgrading packageurl-js will break the way dep-graphs are currently generated. This PR keeps it at an older version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "event-loop-spinner": "^2.0.0",
     "gunzip-maybe": "^1.4.2",
     "mkdirp": "^1.0.4",
-    "packageurl-js": "^1.1.1",
+    "packageurl-js": "^1 < 1.2.1",
```
Comment (gergo-papp) [inline package.json:45]: why not pin it to the last good version?
```suggestion
    "packageurl-js": "1.2.0",
```

File: package.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
     "event-loop-spinner": "^2.0.0",
     "gunzip-maybe": "^1.4.2",
     "mkdirp": "^1.0.4",
-    "packageurl-js": "^1.1.1",
+    "packageurl-js": "1.2.0",
```
Comment (gergo-papp) [inline package.json:45]: you also need to update `package-lock.json`

can be done by running `npm install` (and then run `npm ci` just to make sure the lock file is in sync with the package file)

Comment (SteveShani) [reply package.json:45]: I did. It didn't change

Comment (SteveShani) [reply package.json:45]: Since this is a library, I don't think package-lock.json is relevant.

Comment (SteveShani) [reply package.json:45]: either way, it didn't affect packageurl-js (and it has no dependencies)

Comment (gergo-papp) [reply package.json:45]: do we want to allow clients of this lib to use an older version? (I don't think we need to, but if we do, then your original fix was probably better)

Comment (gergo-papp) [reply package.json:45]: we still have a lock file, so you could update it - that will make sure that we run local tests with the desired version

Comment (SteveShani) [reply package.json:45]: I don't think we care all that much about which version they use.

I'll push the lockfile anwyay

File: package-lock.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "snyk-docker-plugin",
```
Comment (gergo-papp) [inline package-lock.json:2]: this doesn't look right, the `packageurl-js` lib did not change

Comment (SteveShani) [reply package-lock.json:2]: I believe that's because the version was already 1.2.0

**Review by gergo-papp — APPROVED** [review]

**Pull request merged**